guides/common/modules/proc_creating-discovery-rules.adoc

```diff
@@ -23,7 +23,7 @@ For example: `cpu_count  > 8`.
 . In the *Hostname* field, enter the pattern to determine host names for multiple hosts.
 This uses the same ERB syntax that provisioning templates use.
 The host name can use the `@host` attribute for host-specific values and the `rand` macro for a random number or the `sequence_hostgroup_param_next` macro for incrementing the value.
-For more information about provisioning templates, see xref:provisioning-templates_{context}[] and the API documentation.
+For more information about provisioning templates, see xref:provisioning-templates[] and the API documentation.
```

Comment:
```suggestion
For more information about provisioning templates, see xref:provisioning-templates[].
```

out of scope but I'd drop this. It was originally probably meant as a reference to the DSL documentation: https://FQDN/templates_doc which AFAIK helps writing ERB templates